training/distributed_training/pytorch/model_parallel/gpt2/smp-train-gpt-simple.ipynb

```diff
@@ -76,7 +76,7 @@
     "\n",
```

Comment:
<p>Again, not sure if we really need to have the Experiments trial, even for CI. If not needed, this makes the notebook to have an unnecessary step and overly complicated. </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3311/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

Reply:
<p>I think you can remove everything but the last line.</p><p> </p><pre class="ql-syntax" spellcheck="false">smp_estimator.fit(inputs=data_channels, logs=True)
</pre>

Reply:
<p>Updated</p>